website_google_tag_manager/views/website_views.xml

```diff
@@ -1,16 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why removing this?

Reply:
En v12 Odoo remove all atributes (analytics, google maps, etc...) in this form. Now this form is just a popup to create a new website.

Reply:
But then you need other form where putting this. There should be a `website` form...

Reply:
Doesn't look like it